katas/content/qubit/index.md

```diff
@@ -153,11 +153,11 @@ The state of a qubit (or, more generally, the state of a quantum system) is defi
 
 In Q#, qubits are represented by the `Qubit` data type. On a physical quantum computer, it's impossible to directly access the state of a qubit, whether to read its exact state, or to set it to a desired state, and this data type reflects that. Instead, you can change the state of a qubit using quantum gates, and extract information about the state of the system using measurements.
 
-That being said, when you run Q# code on a quantum simulator instead of a physical quantum computer, you can use diagnostic functions that allow you to peek at the state of the quantum system. This is very useful both for learning and for debugging small Q# programs.
+Thus, when you run Q# code on a quantum simulator instead of a physical quantum computer, you can use diagnostic functions that allow you to peek at the state of the quantum system. This is very useful both for learning and for debugging small Q# programs.
```

Comment:
This change changes meaning: "thus" means "as a consequence of this", and the previous paragraph says that you cannot access the state. Here, "that being said" disagrees with the previous paragraph and points out an exception. Please revert or rephrase